src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -277,6 +277,14 @@
     crafted: true
 """
 
+helps['functionapp deployment list-publishing-credentials'] = """
+type: command
+short-summary: Get the details for available function app publishing credentials.
+examples:
+  - name: Get the details for available function app deployment publishing credentials.
+    text: az functionapp deployment list-publishing-credentials --name MyFunctionApp   --resource-group MyResourceGroup
```

Comment:
you need to add a `crafted:true` here too

Reply:
added.